epictrack-api/src/api/services/staff.py

```diff
@@ -79,6 +79,14 @@ def update_staff(cls, staff_id: int, payload: dict):
         staff = staff.update(payload)
         return staff
 
+    @classmethod
+    def update_last_active(cls, staff_id: int):
+        """Update staff's last active time."""
+        staff = Staff.find_by_id(staff_id)
+        staff.last_active_at = datetime.now()
```

Comment:
Can you check the base_mode.py. we are updateing the utc in both created_at and updated_at

Reply:
done